lms/static/js/student_account/views/RegisterView.js

```diff
@@ -112,6 +112,10 @@
                     // The form did not get submitted due to validation errors.
                         $(this.el).show(); // Show in case the form was hidden for auto-submission
                     }
+                    if (!_.compact(this.errors).length) {
```

Comment:
We can add else to previous if statement instead of writing a new if.

Reply:
I have updated code